### PR TITLE
RSPEC updated

### PR DESCRIPTION
**Pull request by csaba-sagi-sonarsource**
Updates the RSPECs for CS and VBNET

### CODE REVIEW

File: analyzers/rspec/cs/S1192_c#.html [hunk 1]
```diff
@@ -32,9 +32,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>The following are ignored:</p>
-<ul>
-  <li> literals with fewer than 5 characters </li>
-  <li> literals matching one of the parameter names </li>
-  <li> literals used in attributes </li>
-</ul>
+<p> * literals with fewer than 5 characters</p>
```
Comment (andrei-epure-sonarsource) [inline analyzers/rspec/cs/S1192_c#.html:35]: this shouldn't be included

this happens when there is a space before the `*` in JIRA (yes 🤦‍♂️ , we need to manually remove the spaces )

File: analyzers/rspec/vbnet/S1192_vb.net.html [hunk 2]
```diff
@@ -36,9 +36,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>The following are ignored:</p>
-<ul>
-  <li> literals with fewer than 5 characters </li>
-  <li> literals matching one of the parameter names </li>
-  <li> literals used in attributes </li>
-</ul>
+<p> * literals with fewer than 5 characters</p>
```
Comment (andrei-epure-sonarsource) [inline analyzers/rspec/vbnet/S1192_vb.net.html:39]: same here

File: analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S1192_cs.html [hunk 3]
```diff
@@ -32,9 +32,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>The following are ignored:</p>
-<ul>
-  <li> literals with fewer than 5 characters </li>
-  <li> literals matching one of the parameter names </li>
-  <li> literals used in attributes </li>
-</ul>
+<p> * literals with fewer than 5 characters</p>
```
Comment (andrei-epure-sonarsource) [inline analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S1192_cs.html:35]: same here

File: analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S1192_vb.html [hunk 4]
```diff
@@ -36,9 +36,7 @@ <h2>Compliant Solution</h2>
 </pre>
 <h2>Exceptions</h2>
 <p>The following are ignored:</p>
-<ul>
-  <li> literals with fewer than 5 characters </li>
-  <li> literals matching one of the parameter names </li>
-  <li> literals used in attributes </li>
-</ul>
+<p> * literals with fewer than 5 characters</p>
```
Comment (andrei-epure-sonarsource) [inline analyzers/src/SonarAnalyzer.Utilities/Rules.Description/S1192_vb.html:39]: same here

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**